SoftLayer/CLI/block/order.py

```diff
@@ -56,13 +56,28 @@
                   'storage_as_a_service',
                   'enterprise',
                   'performance']))
+@click.option('--billing',
+              type=click.Choice(['hourly', 'monthly']),
+              default='monthly',
+              show_default=True,
+              help="Optional parameter for Billing rate. Default to monthly")
```

Comment:
show_default=True - removed as per discussion with David. will push latest file in a while.